packages/cloudflare/src/cli/templates/cache-handler/cache-store.ts

```diff
@@ -0,0 +1,65 @@
+import type { IncrementalCacheValue } from "next/dist/server/response-cache";
+
+export type CacheEntry = {
+  lastModified: number;
+  value: IncrementalCacheValue | null;
+};
+
+export type CacheStore = {
+  get: (key: string) => Promise<CacheEntry | null>;
+  put: (key: string, entry: CacheEntry, ttl?: number) => Promise<void>;
+};
+
+export function getCacheStore() {
+  const kvName = process.env.__OPENNEXT_KV_BINDING_NAME;
+  if (process.env[kvName]) {
```

Comment:
I think we should remove the bindings from `process.env` and we should move to `getCloudflareContext` - could be done in a follow up PR, in which case it would be nice to add a `//TODO: ...`

Reply:
yep, you can create a TODO/issue for that. thanks.

Reply:
#137